src/coreclr/jit/llvm.cpp

```diff
@@ -493,6 +481,35 @@ CorInfoType Llvm::getCorInfoTypeForArg(CORINFO_SIG_INFO& sigInfo, CORINFO_ARG_LI
     return strip(corTypeWithMod);
 }
 
+FunctionType* Llvm::getFunctionType()
+{
+    // TODO-LLVM: delete this when these signatures implemented
+    if (_sigInfo.hasExplicitThis() || _sigInfo.hasTypeArg())
+        failFunctionCompilation();
+
+    std::vector<llvm::Type*> argVec(_llvmArgCount);
+    llvm::Type*              retLlvmType;
+
+    for (unsigned i = 0; i < _compiler->lvaTableCnt; i++)
+    {
+        LclVarDsc varDsc = _compiler->lvaTable[i];
```

Comment:
```suggestion
        LclVarDsc* varDsc = _compiler->lvaGetDesc(i);
```
`LclVarDsc`s are always addressed though pointers as they are rather large and expensive to copy.